spring-boot-actuator/pom.xml

```diff
@@ -139,6 +139,11 @@
 		</dependency>
 		<dependency>
 			<groupId>org.infinispan</groupId>
+			<artifactId>infinispan-core</artifactId>
+			<optional>true</optional>
+		</dependency>
```

Comment:
Sorry but that link doesn't give me any extra info. Can you please clarify? What justify the fact we need to add two dependencies now? Having a `infinispan-jcache` module IMO should be enough.